### ISSUE TITLE
How do we set up admins?

### ISSUE THREAD

**Issue by vicky-d1** [emfoundation/ce100-app#71]
Apologies if I've missed this, currently in the designs we don't reflect how an admin would add another admin user. How are you planning on setting up the initial admins?

**Comment by asimonw** [comment]
@vicky-d1 see #4. This should be reflected in the design, which is currently not the case.

**Comment by fannyBkane** [comment]
See 'Add new user' and 'Edit user' states on Zeplin.

**Comment by iteles** [comment]
@fannyBkane How do actually get to the 'Edit User' screen? Where would I come from in the flow of screens?

**Comment by asimonw** [comment]
Not sure Fanny will still see this today, so I can give some indication. 

At the bottom of the screen (on mobile for an admin) you see a '+' button. On tap this should expand to give you three options: Add user, add org or add news item. I'm not sure the expanded state is anywhere on Zeplin at the moment. @fannyBkane could you add it to one of the screens or indicate here where to find it on Zeplin?

**Comment by fannyBkane** [comment]
Admins get to the 'Edit user' screen from the Contact list. See screen
named 'Contact list-open (admin)'

(I have just updated it as the edit icon was missing).

@alex The + button on the bottom nav allows admin to add a new user. See
screen named 'Add new user (admin)'

*Fanny Breteau*
Designer ~ Digital Team

t: +44 (0) 1983 296 463
e: fanny.breteau@ellenmacarthurfoundation.org
<scs@ellenmacarthurfoundation.org>

<http://ellenmacarthurfoundation.org>

**Comment by asimonw** [comment]
@fannyBkane You're right, I was confusing 'Add' and 'Edit' user... Same screen, different flow ;)
